mkdocs.yml

```diff
@@ -0,0 +1,65 @@
+# pip install -r docs/requirements.txt
+# mkdocs serve
+
+site_name: gradle refreshVersions
+site_url: https://jmfayard.github.io/refreshVersions/
+repo_name: refreshVersions
+repo_url: https://github.com/jmfayard/refreshVersions
+site_description: "Dependency Management: from Hell to Paradise"
+site_author: jmfayard LouisCAD
+remote_branch: gh-pages
+#edit_uri: ""
+
+extra:
+  version:
+    gradle: 6.6.1
+    refreshVersions: '0.9.6'
+    refreshVersionsDev: '0.9.6-dev-007'
```

Comment:
Would be cool to have the release GitHub Action automatically update this on the master branch when it's releasing a dev version (but not a snapshot or a release). Do you have any ideas on how it could be done safely enough?

Or do you thinkg it is not worth it, especially since we support self-update discovery?

Reply:
Let's over-engineer other things, adding a line about this in the CONTRIBUTING guide is enough )